smithy-model/src/main/java/software/amazon/smithy/model/node/internal/JsonParser.java

```diff
@@ -199,12 +212,12 @@ private void readArray() throws IOException {
         skipWhiteSpace();
         if (readChar(']')) {
             nestingLevel--;
-            handler.endArray(array);
+            handler.endArray(array, location);
             return;
         }
         do {
             skipWhiteSpace();
-            handler.startArrayValue(array);
+            // handler.startArrayValue(array);
```

Comment:
There are a decent amount of commented out lines here that, if uncommented, would route to methods on `JsonHandler` that no longer exist.

Reply:
This is meant to help us figure out how to update this class if/when the upstream project gets updated